doc/getting_started/local_installation.md

```diff
@@ -22,8 +22,11 @@ Note: Any time you want to stop and restart Pachyderm, you should start fresh wi
 # For OSX:
 $ brew tap pachyderm/tap && brew install pachyderm/tap/pachctl@1.6
 
-# For Linux (64 bit):
+# For Linux (64 bit) or Window 10+ on WSL:
 $ curl -o /tmp/pachctl.deb -L https://github.com/pachyderm/pachyderm/releases/download/v1.6.6/pachctl_1.6.6_amd64.deb && sudo dpkg -i /tmp/pachctl.deb
+
```

Comment:
Looks like there are some extra lines here. Not a huge deal, but would be nice to remove.

Reply:
removed the extra lines.
@dwhitena